package.json

```diff
@@ -26,19 +26,27 @@
     "license": "SEE LICENSE IN LICENSE.txt FILE",
     "dependencies": {
         "@babel/polyfill": "^7.2.5",
-        "@jahia/apollo-dx": "^1.1.0",
+        "@jahia/data-helper": "^0.0.5",
         "@jahia/design-system-kit": "^1.1.4",
         "@jahia/icons": "^1.1.0",
         "@jahia/moonstone": "^0.8.2",
-        "@jahia/react-apollo": "^1.1.0",
-        "@jahia/react-material": "^1.1.3",
+        "@jahia/moonstone": "^0.7.0",
```

Comment:
Is this a git error, or are you adding moonstone 0.7.0 when we already have 0.8.2 ?